lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -22,4 +21,18 @@ function submitGrade({ student, grade, authToken }) {
   });
 }
 
-export { submitGrade };
+/**
+ * Fetches a student's grade from the LTI endpoint
+ *
+ * @param {Object} student - Student object
+ * @param {string} authToken - The auth token from the config
+ * @return {Promise<Object>} - The fetched result
```

Comment:
This is consistent with the above. It would be great to use `@typedef`s to document the shape of the `student` object and result here though.